docs/dev-manuals/adr/0000-use-monolithic-frontend.md

```diff
@@ -0,0 +1,39 @@
+---
+sidebar_position: 1
+---
+
+# Use Monolithic Frontend
+
+## Context and Problem Statement
+
+We consider two different ways of building our frontend: either a micro frontend, or a monolithic frontend.
+
+## Considered Options
+
+- micro frontend
+- monolithic frontend
+
+## Decision Outcome
+
+Chosen option: "monolithic frontend", because
+
+- with the given resources (5 developers) and experiences, a monolithic frontend is easier to implement and thus allows us to focus on more important aspects of the reference architecture
+
+### Consequences
+
+- Good, because we do not need to consider requirements of a micro frontend with respect to the API gateway
```

Comment:
What's the difference between consequences and pros and cons?

Reply:
Tbh I do not know precisely, the template description is not that helpful.
In some cases I just mentioned stuff twice, sometimes I tried to extend the stuff in pro/con a bit in the consequences stuff (the more this results in part)
But I would also be fine if we decide to remove the consequences section completely, and instead put all the important stuff in pro/con

Reply:
Sounds good!

Reply:
so, ok as it is, or shall we remove the consequences section?

Reply:
I would not remove it entirely. Idk exactly but I think pros and cons solely refer to what is good and what is bad regarding the AD. Consequences on the other hand are mainly about what will definitely happen if we make that AD -- consequences of course can also be categorized as good or bad and yes, most likely most of the pros and cons will make it into the consequences section.
Thinking about this, how about we follow your suggestion, @nk-coding, and "put all the important stuff in pro/con" but we do not remove the consequences section and instead write something like "The consequences can be found in the pro/con section".

Reply:
The standard/template says that the consequences section should also contain good/bad stuff (Starting with good, because...)
I would not add the reference, I hope this is clear (because why would we mentioned it below otherwise)